src/expressions/if-expr.md

```diff
@@ -111,6 +111,27 @@ match EXPR {
 }
 ```
 
+If the expression is a [lazy boolean operator expression][_LazyBooleanOperatorExpression_] then
```

Comment:
This is a very roundabout way of saying "The expression cannot be a lazy boolean operator expression."

Reply:
Changed this.